Client/WebFilters/AdblockServicePublisher.swift

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveCore
+
+class AdblockServicePublisher {
+  /// Ad block service used for getting certain components
+  private weak var adBlockService: AdblockService?
+  
+  /// The published results
+  @Published var filterListResults: [String: (date: Date, result: Result<URL, Error>)]
```

Comment:
can be `private(set)` I guess?

Reply:
Must be a comment on old code. This doesn't exist any longer